### DISCUSSION TITLE
Does TabularPredictor Support Incremental Learning or Expanding Training with Additional Data?

### DISCUSSION THREAD

**Discussion by leeprinxin** [autogluon/autogluon#4403] [General]
Can TabularPredictor support incremental learning? I have a requirement where, over time, I want to add future data for tuning without retraining from scratch. Currently, Multimodal mentions expanding training (https://auto.gluon.ai/stable/tutorials/multimodal/advanced_topics/continuous_training.html#use-case-1-expanding-training-with-additional-data-or-training-time), but why doesn't TabularPredictor provide this feature?

**Comment by Innixma** [comment]
TabularPredictor does not support incremental training. Tree models such as XGBoost, CatBoost, LightGBM don't support incremental training fundamentally, and trying to add support within Tabular would be too much of a hassle for marginal benefit. We recommend the user to re-train from scratch with all available data for the best results.
